configurations/create_imaqal_database.py

```diff
@@ -0,0 +1,187 @@
+from src.pipeline_configuration_spec import *
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="CREATE-IMAQAL-POOL",
+    description="Creates the initial Imaqal Pool from demographics responses to IMAQAL, IMAQAL_COVID19, "
+                "SSF-ELECTIONS, SSF-DCF, SSF-SLD, SSF-REC, and SSF-PPE.",
+    engagement_database=EngagementDatabaseClientConfiguration(
```

Comment:
add     project_start_date=isoparse("2020-01-01T00:00:00+03:00"), and     project_end_date=end of ssf elections so that we can filter data outside the period we agreed on

Reply:
Note that this doesn't affect which data gets synced from rapid pro

Reply:
Noted.